driven/storage/adapter_surveys.go

```diff
@@ -36,14 +36,17 @@ func (a *Adapter) GetSurvey(id string, orgID string, appID string) (*model.Surve
 }
```

Comment:
We need to make sure that the users cannot update the `sensitive` or `anonymous` fields on the survey after they are created